mirage/views/search.ts

```diff
@@ -66,6 +100,21 @@ const resourceMetadataByType: Partial<Record<OsfmapResourceTypes, any>> = {
             '@value': faker.lorem.sentence(),
             '@type': rdfString,
         }],
+        funder: [_shareOrganizationField()],
+        hasOsfAddon: [
+            {
+                prefLabel: [{
+                    '@value': 'OSF Storage', // CR question: do we need to include osf storage as addons?
+                    '@type': rdfString,
+                }],
+            },
+            {
+                prefLabel: [{
+                    '@value': 'Box',
+                    '@type': rdfString,
```

Comment:
sidenote: oof, all these `"@type": "http://www.w3.org/1999/02/22-rdf-syntax-ns#string"` are not helpful -- does any frontend code actually use them (i see it only in mirage), or would they be safe to remove?

Reply:
They are safe to remove. I was thinking a yet-to-be-written utility function (something like `getRefProperty` or `getStringProperty`) may want to be aware of the `@type`, but as of right now, I agree that they are just adding clutter